core/executor/src/wasm_executor.rs

```diff
@@ -174,36 +200,37 @@ impl_function_executor!(this: FunctionExecutor<'e, E>,
 				HexDisplay::from(&key)
 			);
 		}
-		let storage_key = ChildStorageKey::from_vec(storage_key)
-			.ok_or_else(||
-				UserError("ext_set_child_storage: child storage key is invalid")
-			)?;
-		this.ext.set_child_storage(storage_key, key, value);
+		this.with_subtrie(&storage_key[..], |this, subtrie|
+			this.ext.set_child_storage(&subtrie, key.clone(), value.clone())
+		).expect("Called from a valid SubTrie instance");
 		Ok(())
 	},
-	ext_clear_child_storage(storage_key_data: *const u8, storage_key_len: u32, key_data: *const u8, key_len: u32) => {
-		let storage_key = this.memory.get(
-			storage_key_data,
-			storage_key_len as usize
-		).map_err(|_| UserError("Invalid attempt to determine storage_key in ext_clear_child_storage"))?;
-		let key = this.memory.get(key_data, key_len as usize).map_err(|_| UserError("Invalid attempt to determine key in ext_clear_child_storage"))?;
+	ext_clear_child_storage(
+		prefixed_storage_key_data: *const u8,
```

Comment:
In some places, `storage_key_data` was renamed to `prefixed_storage_key_data` and in some others it is renamed to `keyspace_data`. On the other hand, in some places it is left as `storage_key_data`.

Why is this difference there? Can we use one name?